### ISSUE TITLE
`dials.image_viewer` does not work in `conda` build

### ISSUE THREAD

**Issue by dennisbrookner** [dials/dials#2109]
I'm (excitedly) using my new conda dials installation
```
(dials) dennisbrookner@dhcp-10-250-200-254 dials_new % dials.version
DIALS 3.10.dev1
Python 3.9.12
Installed in: /Users/dennisbrookner/miniconda3/envs/dials/lib/python3.9/site-packages/dials
```
And I've for the first time encountered something that doesn't work: 
```
(dials) dennisbrookner@dhcp-10-250-200-254 dials_new % dials.image_viewer integrated.expt integrated.refl 
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
The following parameters have been modified:

input {
  experiments = integrated.expt
  reflections = integrated.refl
}

This program needs access to the screen. Please run with a
Framework build of python, and only when you are logged in
on the main display of your Mac.
```
I suppose it is possible that this is an issue simply with the newest version of dials and not with the conda build specifically, but it smells like a conda thing to me.

I'm on an M1 2020 MacBook Pro running Big Sur 11.2.3.

**Comment by dennisbrookner** [comment]
After a little bit of googling, this seems to be related to the difference between `python` and `pythonw`.

**Comment by ndevenish** [comment]
Ah, we fixed this issue with the downloadable release bundle, but since we're using plain python entry_points on the conda distribution, this has reoccured. I _believe_ the solution is the difference between `console_scripts` and `gui_scripts` but there are a few differences with the conda build process which might make this complicated 🤔 

Replacing the shebang with `pythonw` is _a_ solution, but then you run into the fact that the `dials.image_viewer` is completely broken on the conda package (e.g. wx complains, then python segfaults). So this is something to fix before we release 3.10 proper (or in the worst case, distribute without the image viewer initially).

**Comment by ndevenish** [comment]
e.g. 
```
$ pythonw $(which dials.image_viewer) $(dials.data get -q thaumatin_i04)
```
Gives:
```
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
The following parameters have been modified:

input {
  experiments = <image files>
}

Fatal Python error: Segmentation fault

Current thread 0x0000000104ebfd40 (most recent call first):
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/site-packages/dials/util/image_viewer/spotfinder_frame.py", line 263 in setup_toolbar
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/site-packages/dials/util/image_viewer/slip_viewer/frame.py", line 129 in __init__
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/site-packages/dials/util/image_viewer/spotfinder_frame.py", line 128 in __init__
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/site-packages/dials/util/image_viewer/spotfinder_wrap.py", line 64 in display
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/site-packages/dials/command_line/image_viewer.py", line 178 in show_image_viewer
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/site-packages/dials/command_line/image_viewer.py", line 213 in run
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/lib/python3.10/contextlib.py", line 79 in inner
  File "/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/bin/dials.image_viewer", line 11 in <module>

....
/private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/bin/pythonw: line 3: 78799 Segmentation fault: 11  /private/var/folders/yq/n9kwymws3q317f02dwpl9jrr0000gn/T/tmp.L3sK4ZvM/ENV/python.app/Contents/MacOS/python "$@"
```

**Comment by dennisbrookner** [comment]
I totally forgot about this, but I just confirmed that is has been fixed as of the `3.15` conda build of dials. Thanks devs!!
